keps/1-kubernetes-enhancement-proposal-process.md

```diff
@@ -221,106 +218,34 @@ Metadata items:
 
 ### KEP Workflow
 
-TODO(jbeda) Rationalize this with status entires in the Metadata above.
-
-A KEP is proposed to have the following states
-
-- **opened**: a new KEP has been filed but not triaged by the responsible SIG or
-  working group
-- **accepted**: the motivation has been accepted by the SIG or working group as in
-  road map
-- **scoped**: the design has been approved by the SIG or working group
-- **started**: the implementation of the KEP has begun
-- **implemented**: the implementation of the KEP is complete
-- **deferred**: the KEP has been postponed by the SIG or working group despite
-  agreement on the motivation
-- **superseded**: the KEP has been superseded by another KEP
-- **retired**: the implementation of the KEP has been removed
-- **rejected**: the KEP has been rejected by the SIG or working group
-- **orphaned**: the author or developer of the KEP is no longer willing or able
-  to complete implementation
-
-with possible paths through the state space
-
-- opened -> deferred (a)
-- opened -> rejected (b)
-- opened -> orphaned (c)
-- opened -> accepted -> orphaned (d)
-- opened -> accepted -> scoped -> superseded (e)
-- opened -> accepted -> scoped -> orphaned (f)
-- opened -> accepted -> scoped -> started -> retired (g)
-- opened -> accepted -> scoped -> started -> orphaned (h)
-- opened -> accepted -> scoped -> started -> superseded (i)
-- opened -> accepted -> scoped -> started -> implemented (j)
-- opened -> accepted -> scoped -> started -> implemented -> retired (k)
-
-the happy path is denoted by (j) where an KEP is opened; accepted by a SIG as in
-their roadmap; fleshed out with a design; started; and finally implemented. As
-Kubernetes continues to mature, hopefully metrics on the utilization of features
-will drive decisions on what features to maintain and which to deprecate and so
-it is possible that a KEP would be retired if its functionality no longer provides
-sufficient value to the community.
+A KEP is has the following states
```

Comment:
thanks for reducing these states; I do still think an `Orphaned` state would be useful but we can add it later as needed I suppose. I think the work done to add statuses to issues/pull requests would suggest that adding a `Started` state would be helpful

Reply:
s/is has/has